### PR TITLE
Add Streamlit Groq demo app

### PR DESCRIPTION
**Pull request by tonykipkemboi**
This is a simple Streamlit chatbot powered by Groq! 

You can switch between different models and adjust token limits. 

As more models are added, adding them to the dictionary and doing slight code modifications to interact with them on the chat interface is straightforward.

I have also added a video tutorial using the same code base in the readme file.

### CODE REVIEW

File: groq_streamlit_demo/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+**Check out the video tutorial 👇**
+
+<a href="https://youtu.be/WQvinJGYk90">
+  <img src="https://img.youtube.com/vi/WQvinJGYk90/hqdefault.jpg" alt="Watch the video" width="100%">
+</a>
+
+## Features
+
+- **Model Selection**: Users can select between `mixtral-8x7b-32768` and `llama2-70b-4096` models to tailor the conversation according to the capabilities of each model.
```
Comment (hozen-groq) [inline groq_streamlit_demo/README.md:19]:
```suggestion
- **Model Selection**: Users can select between `mixtral-8x7b-32768`, `llama2-70b-4096`, and `gemma-7b-it` models to tailor the conversation according to the capabilities of each model.
```

Comment (hozen-groq) [reply groq_streamlit_demo/README.md:19]: Feel free to ignore this if the Streamlit app itself doesn't support Gemma!

**Comment by hozen-groq** [comment]
Just two suggestions:

- To mention the https://console.groq.com link rather than the https://groq.com link
- To add Gemma if it is supported since we do support it via the API

Other than that, this looks fantastic! Thank you so much for contributing! 🤩

File: groq_streamlit_demo/README.md (hunk 1 above)
Comment (tonykipkemboi) [reply groq_streamlit_demo/README.md:19]: Awesome! Yes, it does support Gemma. Thanks for catching that. I added Gemma but never updated the README accordingly.

**Comment by tonykipkemboi** [comment]
Awesome! Thank you for the corrections. I've committed to your suggestions.

**Pull request merged**